src/main/java/seedu/recipe/logic/LogicManager.java

```diff
@@ -55,8 +55,8 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
     }
 
     @Override
-    public ReadOnlyAddressBook getAddressBook() {
-        return model.getAddressBook();
+    public ReadOnlyRecipeBook getAddressBook() {
```

Comment:
Some occurences of AddressBook (method names) not refactored, but I've handled it in my PR so its merge-ready